clubs/mixins.py

```diff
@@ -0,0 +1,241 @@
+import datetime
+from collections import OrderedDict
+
+from django.core.exceptions import FieldDoesNotExist, MultipleObjectsReturned, ObjectDoesNotExist
+from django.db.models import BooleanField, ManyToManyField
+from rest_framework import serializers
+from rest_framework.renderers import JSONRenderer
+from rest_framework.response import Response
+from rest_framework.utils.serializer_helpers import ReturnDict, ReturnList
+
+
+class ManyToManySaveMixin(object):
+    """
+    Mixin for serializers that saves ManyToMany fields by looking up related models.
+
+    In the Meta class, create a new attribute called "save_related_fields" in the Meta
```

Comment:
two "in the Meta class" clauses in the sentence, can remove one or the other ;)

Reply:
Nice catch! :sweat_smile: